tests/Browser/UsuarioTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Tests\Browser;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use App\User;
+
+class UsuarioTest extends DuskTestCase
+{
+    public function testCreateUser() {
+        $this->newUser = factory(User::class)->create();
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::find(1))
+                    ->visit('/usuario/create')
+                    ->type('nombre', $this->newUser->nombre)
+                    ->type('apellido', $this->newUser->apellido)
+                    ->type('usuario', $this->newUser->usuario)
+                    ->type('password', $this->newUser->password)
+                    ->press('Guardar')
+                    ->assertDontSee('required');
+        });
+    }
+
+    public function testUpdateUser(){
+          $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::find(1))
+                    ->visit('/usuario')
+                    ->click('.glyphicon-edit')
+                    ->press('Guardar')
+                    ->assertDontSee('required');
+            });
+    }
+
+    public function testDeleteUser(){
+      $this->newUser = factory(User::class)->create();
```

Comment:
este puede ser una variable local (sin el $this->), no tiene que ser global, definila dentro del `function(Browser...` (linea 39)

Reply:
Creo que le puse el $This porque sino me tiraba error, como que no existia la variable newUser. ahora vuelvo a probar.